{{cookiecutter.app_name}}/.tool-versions

```diff
@@ -1 +1,2 @@
-golang 1.20
+golang 1.20{% if cookiecutter.use_openapi == "yes" %}
```

Comment:
The formatting is a bit odd. How about:

```
golang 1.20
{% if cookiecutter.use_openapi == "yes" %}
nodejs 18.15.0
{% endif %}
````

or:

```
golang 1.20
{% if cookiecutter.use_openapi == "yes" %}nodejs 18.15.0{% endif %}
````

Reply:
As [Cookiecutter template](https://github.com/cookiecutter/cookiecutter#for-template-creators) uses [Jinja2](https://jinja.palletsprojects.com/en/2.11.x/templates/) we could add `-` to the first option to remove extra line break

```go
golang 1.20
{% if cookiecutter.use_openapi == "yes" -%}
nodejs 18.15.0
{%- endif %}
```